src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -53,6 +53,22 @@ class HealthCertificateService {
 	private(set) var testCertificateRequests = CurrentValueSubject<[TestCertificateRequest], Never>([])
 	private(set) var unseenTestCertificateCount = CurrentValueSubject<Int, Never>(0)
 
+	var nextValidityTimer: Timer?
+
+	var processNextFireTimestamp: Date? {
```

Comment:
Didn't really get the function name here, can it be changed?

Reply:
i changed it it `var nextFireDate: Date?` - I guess this will be a better name.